src/layouts/product-integration-layout/index.tsx

```diff
@@ -126,18 +127,102 @@ export default function ProductIntegrationLayout({
 					<meta name="robots" content="noindex, nofollow" />
 				</HashiHead>
 			)}
-			<Header
-				productSlug={currentProduct.slug}
+
+			<ContentHeaderCard
 				className={s.header}
-				integration={integration}
-				activeRelease={activeRelease}
-				getVersionChangedURL={getVersionChangedURL}
-				onInstallClicked={() => {
-					console.log('TODO, probably remove this')
-				}}
+				icon={currentProduct.slug !== 'sentinel' ? currentProduct.slug : null}
+				title={integration.name}
+				attribution={`@${integration.organization.slug}`}
+				description={integration.description}
+				note={lastUpdatedString(integration.updated_at)}
+				badges={GetIntegrationBadges(integration, true)}
+				dropdown={
+					integration.versions.length > 1
+						? {
+								text: versionString(
+									activeRelease.version,
+									integration.versions
+								),
+								items: integration.versions
+									.filter((e: string) => e !== activeRelease.version)
+									.map((version: string) => {
+										return {
+											text: versionString(version, integration.versions),
+											href: getVersionChangedURL(version),
+										}
+									}),
+						  }
+						: undefined
+				}
+				links={[
+					{
+						text: 'GitHub',
+						href: integration.subdirectory
+							? `${integration.repo_url}/tree/main${integration.subdirectory}`
+							: integration.repo_url,
+						icon: <IconGithub16 />,
+					},
+				]}
 			/>
+
 			<h1>{title}</h1>
 			<div className={className}>{children}</div>
 		</SidebarSidecarLayout>
 	)
 }
+
+function lastUpdatedString(updatedAtString: string) {
```

Comment:
Personal preference: This would be nice to extract in a utils file within the integrations layout dir.

Reply:
Hah if we do that, I might want to harden this a bit more 😆 this gets the job done for now for integrations, but is currently probably not 100% accurate (e.g. 5 weeks ago is 1 months ago, regardless of how far into the current month you are).

Reply:
I'd like to see this moved too -- not to `src/lib`, but to a new folder in `src/layouts/product-integration-layout/`. This file is getting pretty long, so it'd help reduce the overwhelm factor when opening this file. 😊

Reply:
Yep, exactly @ashleemboyer I'm not suggesting in the global utils folder but often we will make a local utils file: `src/layouts/product-integration-layout/utils.ts`